src/app/pages/things/channels/channels.component.ts

```diff
@@ -68,45 +66,19 @@ export class ChannelsComponent implements OnInit {
   source: LocalDataSource = new LocalDataSource();
   channels: Channel[];
 
-  gwDataChanNumber = 0;
-  gwCtrlChanNumber = 0;
-  loraChanNumber = 0;
   totalChanNumber = 0;
 
   offset = 0;
-  limit = 20;
+  limit = 100;
```

Comment:
It could be better to set this from a constant...
I know we will change this component in future, but constant is better